packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -514,6 +514,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
               onMouseMove: this._onItemMouseMove.bind(this, item)
             } }
             label={ item.text }
+            onRenderLabel={ this._renderCustomLabel.bind(this, item) }
```

Comment:
Use `@autobind`. Also, `_onRenderLabel` might be a better name to match existing naming conventions in this file